lib/radius.js

```diff
@@ -449,6 +449,9 @@ Radius.decode_attributes = function(data, attr_hash, vendor, raw_attrs) {
     length = data.readUInt8(1);
     value = data.slice(2, length);
 
+    if (!length)
+      throw new Error("readUInt8(1) has length of 0");
```

Comment:
I'm sorry.. I just saw your post after I posted mine. I can change the error message. No problem!